shared/js/background/classes/score.es6.js

```diff
@@ -62,6 +62,8 @@ class Score {
                     },
                     message: message
                 }
+
+                return result
```

Comment:
Lint complains about assignments in `return` statements. See https://eslint.org/docs/rules/no-return-assign for the motivation behind it.